services/adminService.js

```diff
@@ -63,7 +67,7 @@ class AdminService extends main_service{
     AdminCreation = async (data) => {
         try {
             let findAdmin = await this.user.find({ role: "admin" });
-            if (findAdmin.length >= 1 && findAdmin != null) {
+            if ( findAdmin.length >= 1 && findAdmin != null) {
```

Comment:
userAdmin/adminUser and condition could be userAdmin?.length ( check option chaining)